package-lock.json

```diff
@@ -5892,9 +5892,9 @@
 			}
 		},
 		"lodash": {
-			"version": "4.17.11",
-			"resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.11.tgz",
-			"integrity": "sha512-cQKh8igo5QUhZ7lg38DYWAxMvjSAKG0A8wGSVimP07SIUEK2UO+arSRKbRZWtelMtN5V0Hkwh5ryOto/SshYIg==",
+			"version": "4.17.15",
+			"resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.15.tgz",
+			"integrity": "sha512-8xOcRHvCjnocdS5cpwXQXVzmmh5e5+saE2QGoeQmbKmRS6J3VQppPOIt0MnmE+4xlZoumy0GPG0D0MVIQbNA1A==",
```

Comment:
Why we still keep the old version in the package.json file? 

https://github.com/LiskHQ/lisk-sdk/blob/a898540c67b5e601020c964889a5d64c3b54b64a/framework/package.json#L73

Reply:
There are other libraries which are depending on `^4.17.11` version. Can we make sure all those dependence are also updated.

Reply:
apart from `framework` others are a transient dependency. so if you run `npm audit` in the root it won't complain. so should be good. for lisk-core I will update them as well.